cli/src/main/java/io/specmesh/cli/KafkaProvisionCommand.java

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright 2023 SpecMesh Contributors (https://github.com/specmesh)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.specmesh.cli;
+
+import static picocli.CommandLine.Command;
+
+import io.confluent.kafka.schemaregistry.client.CachedSchemaRegistryClient;
+import io.confluent.kafka.schemaregistry.client.SchemaRegistryClientConfig;
+import io.specmesh.apiparser.AsyncApiParser;
+import io.specmesh.kafka.KafkaApiSpec;
+import io.specmesh.kafka.Provisioner;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+import org.apache.kafka.clients.admin.Admin;
+import org.apache.kafka.clients.admin.AdminClient;
+import org.apache.kafka.clients.admin.AdminClientConfig;
+import picocli.CommandLine.Option;
+
+/** SpecMesh Kafka Provisioner */
+@Command(name = "provision", description = "Applied provided specification file to the cluster")
+public class KafkaProvisionCommand implements Runnable {
+
+    @Option(
+            names = {"-br", "--brokerUrl"},
+            description = "Cluster url")
+    private String brokerUrl;
+
+    @Option(
+            names = {"-sr", "--schemaRegistryUrl"},
+            description = "schemaRegistryUrl")
+    private String schemaRegistryUrl;
+
+    @Option(
+            names = {"-srApiKey", "--srApiKey"},
+            description = "srApiKey")
+    private String srApiKey;
```

Comment:
Standard argument naming on cli tools is for the `--` prefixed option to be the long name and `-` option to a the short name consisting of only a few characters.

You've got them both the same...

I'd suggest either removing the `-` option or defining an actual short name for them.

Reply:
ok